libr/debug/p/native/linux/linux_debug.c

```diff
@@ -509,3 +514,1397 @@ RList *linux_desc_list (int pid) {
 	closedir (dd);
 	return ret;
 }
+
+
+/* Coredump  functions */
+
+static map_file_t mapping_file = {0,0};
+
+static char *prpsinfo_get_fname(FILE *f) {
+        char *p;
+        int c;
+        int pos;
+
+        p = malloc(16);
+        pos = 0;
+        while ((c = fgetc(f)) != EOF && c != '\0')
+                p[pos++] = c;
+
+        if (c == '\0')
+                p[pos] = c;
+
+        return p;
+}
+
+static char *get_basename(char *pfname, int len) {
+        char *p;
+
+        for (p = pfname + len; p != pfname ; p--)
```

Comment:
`for (p = pfname + len; p != pfname; p--)` (spacing) also use braces here, and at the very least put the `return` on the same line as the `if`